query.go

```diff
@@ -1243,6 +1243,11 @@ func (s *ChainService) GetCFilter(blockHash chainhash.Hash,
 			if !ok {
 				// Query has finished, if we have a result we'll
 				// return it.
+				if resultFilter == nil {
+					return nil, fmt.Errorf("unable to " +
```

Comment:
Would it be useful to extract the error into a variable?

Reply:
Fixed: https://github.com/lightninglabs/neutrino/pull/187/commits/8c1f6cf2a794a7259b097602469cf0f01d2d49bb